reesegreen2014/dog-info-hub
# Feature/homepage header footer

#### What does this PR do?
- This PR adds and styles the homepage, header, and footer components of the Dog Info Hub application. It includes a responsive design for different devices, integrates a menu bar for the header, integrates social media links in the footer, and ensures the footer stays at the bottom of the page.
#### Where should the reviewer start?
1. Start by reviewing the HomePage, Header, and Footer components in their respective files:
- `src/components/HomePage/HomePage.js`
- `src/components/Header/Header.js`
- `src/components/Footer/Footer.js`
2. Then, check the corresponding CSS files for each component to ensure styles are correctly applied:
- `src/components/HomePage/HomePage.css`
- `src/components/Header/Header.css`
- `src/components/Footer/Footer.css`
#### How should this be manually tested?
1. Run the application locally using `npm start`.
2. Navigate to the homepage and verify that:
- The header is correctly displayed with navigation links.
- The homepage content is centered and responsive.
- The footer contains social media links that redirect to the appropriate profiles.
- The footer stays at the bottom of the page on all devices.
#### Any background context you want to provide?
- This update is part of the initial setup for the Dog Info Hub application. The goal is to establish a clean and responsive layout for the main components of the application before adding more functionality and additional pages.
#### What are the relevant tickets (if any)?
- Relevant issues within the Project Board will be closed upon successful completion of merge to main. 
#### Screenshots (if appropriate)
![image](https://github.com/user-attachments/assets/13962ee7-2f46-4390-9611-b6186477dcd7)
![Screen Recording 2024-07-25 at 3 30 38 PM](https://github.com/user-attachments/assets/61d26ecd-b6aa-4553-9784-1f1ede3c8be0)
